ui/pages/Dashboard/components/ProjectsTable.jsx

```diff
@@ -55,6 +56,17 @@ const COLUMNS = [
     width: 1,
     format: project => <CategoryIndicator project={project} />,
   },
+  {
+    name: 'anvil',
+    width: 1,
+    content: 'AnVIL',
+    format: project => (
+      <div>
+        {project.workspaceName &&
+        <Popup content={`AnVIL workspace: ${project.workspaceNamespace}/${project.workspaceName}`} position="top center" trigger={<Icon name="fire" />} />}
```

Comment:
I really like this!

Reply:
this column should only show up if TERRA_API_ROOT_URL is configured (i.e. if `googleLoginEnabled` is true in the state)

Reply:
Updated.

Reply:
If `TERRA_API_ROOT_URL` is not configured the whole `AnVIL` column won't be displayed.